### ISSUE TITLE
traefik reverse proxy with the file backend

### ISSUE THREAD

**Issue by GotekSC** [pyload/pyload#3378]
### Type of ticket:

- [ ] Plugin request
- [ ] Feature request
- [ ] Bug report
- [ x] Question
- [ ] Other

### Summary:

So I want to reverse proxy pyload with traefik, but I get this screen:
```
Error 404: Not Found

Sorry, the requested URL 'http://example.domain.com/pyload' caused an error:

Not found: '/pyload'
```
I tried some things from #3073, like setting the pathprefix in pyload itself, to no avail.

This is the `pyload.toml` I use in traefik.
```
# pyload
[backends]
  [backends.backend-pyload]
    [backends.backend-pyload.servers]
      [backends.backend-pyload.servers.server-pyload-ext]
        url = "http://examplehost:8668/"
        weight = 0

[frontends]
  [frontends.frontend-pyload]
    backend = "backend-pyload"
    passHostHeader = true

    [frontends.frontend-pyload.routes]
          [frontends.frontend-pyload.routes.route-pyload-ext]
        rule = "Host:example.domain.com; PathPrefix: /pyload"

    [frontends.frontend-pyload.headers]
      SSLRedirect = true
      STSSeconds = 315360000
      browserXSSFilter = true
      contentTypeNosniff = true
      forceSTSHeader = true
      SSLHost = "example.domain.com"
      STSIncludeSubdomains = true
      STSPreload = true
      frameDeny = true
```
I tried without the frontends.frontend-pyload.headers, but that also made no difference.

**Comment by GammaC0de** [comment]
First of all, have you followed the instructions in https://github.com/pyload/pyload/issues/3283 and set the path prefix config option?

**Comment by GotekSC** [comment]
Like I mentioned above, I tried suggestions from #3073, which among other things mentioned the path prefix, yes. it's set to `/pyload`

Is #3283 relevant when I cloned from master only a month ago?

**Comment by GammaC0de** [comment]
You don't clone from master, you clone from stable..

**Comment by GotekSC** [comment]
Alright. I deleted the directory, cloned again and checked out from stable. Made sure the path prefix is still set to `/pyload`.

When logged out `../pyload` now correctly redirects to `../pyload/login`. But no css loads. However, I can see umatrix listing 5 in the css column (of course I made sure it's allowed).
When clicking on any of the header links (queue, collector, config etc.) redirection does not work.

On another note: It might be worth updating the wiki pages. Even just a redirection notice to the readme.md would suffice, I guess. I instinctively sought out the wiki page for installation on linux, was confused by the list of dependencies being outdated and ended up following https://wiki.ubuntuusers.de/pyLoad/ with a bit of trial & error.
